docs/source/user/ocr.rst

```diff
@@ -160,3 +166,29 @@ Supported strategies are:
 
 .. note:: When omitted, ``ocr_and_text`` value is used. If you have performance issues, it's worth using the ``auto`` option
 instead as only documents with barely no text will go through the OCR process.
+
+
+OCR Page Seg Mode
+----------------------------
```

Comment:
Please make the length identical...

It should be:

```
OCR Page Seg Mode
-----------------
```

Reply:
It's not solved. Exact code should be:

```
OCR Page Seg Mode
-----------------
```

And not:

```
OCR Page Seg Mode
----------------------------
```

Reply:
Sorry, forgot to commit.